README.md

```diff
@@ -0,0 +1,71 @@
+# Flocx-ui
+
+Flocx plugin for horizon to show and create bids, offers, and contracts
+
```

Comment:
Provide a link to the FLOCX Overview document here so that the contributors understand the scope of the project.